tcms/testruns/templates/testruns/mutable.html

```diff
@@ -95,6 +95,59 @@
                 {{ form.build.errors }}
             </div>
         </div>
+        <div class="form-group">
```

Comment:
I just realized the 2 new fields for planning should be added in testruns/get.html above the existing ones. Use `calendar-o` icon for both of them.